packages/contracts-sdk/src/lib/contracts-sdk.ts

```diff
@@ -114,7 +117,13 @@ export class LitContracts {
   connected: boolean = false;
   isPKP: boolean = false;
   debug: boolean = false;
-  network: 'cayenne' | 'manzano' | 'habanero' | 'custom' | 'localhost';
+  network:
```

Comment:
This mirrors `LIT_NETWORKS_KEYS` from our types - while we're touching these in-lined unions, it would be nice to just change it to reference that so the next time we add a network we don't have as many places to update.